com.unity.render-pipelines.universal/Editor/Lighting/UniversalRenderPipelineLightUI.PresetInspector.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using UnityEngine;
+using UnityEngine.Rendering;
+using UnityEngine.Rendering.Universal;
+
+namespace UnityEditor.Rendering.Universal
+{
+    using CED = CoreEditorDrawer<UniversalRenderPipelineSerializedLight>;
+    using Styles = UniversalRenderPipelineLightUI.Styles;
+
+    static class UniversalRenderPipelineLightUIPreset
+    {
+        [URPHelpURL("light-component")]
+        enum Expandable
+        {
+            General = 1 << 0,
+            Emission = 1 << 1
+        }
+
+        static readonly ExpandedState<Expandable, Light> k_ExpandedState = new ExpandedState<Expandable, Light>(~(-1), "URP-preset");
```

Comment:
Same as the one in the `UniversalRenderPipelineCameraUIPreset`

Reply:
no overlap

Reply:
```suggestion
        static readonly ExpandedState<Expandable, Light> k_ExpandedState = new ExpandedState<Expandable, Light>(0, "URP-preset");
```
~(-1) == 0

Reply:
Thanks Sven! It was applied in latest commit.